packages/ui-components/src/modal/Modal.tsx

```diff
@@ -24,7 +27,10 @@ function Modal(props: ModalProps) {
     appSelector = 'root',
     header = true,
     title,
+    className,
```

Comment:
Thanks for making this more customisable!

Reply:
;)